addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10056,7 +10056,9 @@ msgctxt "#19239"
 msgid "Starting background threads"
 msgstr ""
 
-#empty string with id 19240
+msgctxt "#19240"
+msgid "RePlay"
```

Comment:
I don't think the capital P is okay here. Even this might be a proper nound for Zattoo, Kodi UI only contains 'generic' strings.